tests/Controller/CheckoutRequestControllerTest.php

```diff
@@ -59,4 +60,25 @@ public function testMisconfigurationThrowsException()
 
         self::assertSame(500, $response->getStatusCode());
     }
+
+    /**
+     * @covers NYPL\Services\Controller\CheckoutRequestController::reassignPartnerBarcode()
+     */
+    public function testReassignPartnerBarcode()
+    {
+        putenv('PATRON_BARCODES_70620917062091=12345,678910');
```

Comment:
Remove the puten() line and add the env variable to tests/Mocks/MockConfig.php instead. This will keep the environment variables for the test in one place. The test's pass/fail result will be the same.

Reply:
Fixed